railib/api.py

```diff
@@ -292,6 +290,8 @@ def get_transaction(ctx: Context, id: str) -> dict:
 def get_transaction_metadata(ctx: Context, id: str) -> dict:
     return _get_resource(ctx, f"{PATH_TRANSACTIONS}/{id}/metadata")
 
+def list_transactions(ctx: Context) -> list:
+    return _get_resource(ctx, f"{PATH_TRANSACTIONS}", key="transactions")
```

Comment:
If you want to return a list I would suggest to call `_list_collection` function. Please see https://github.com/RelationalAI/rai-sdk-python/pull/73. I renamed the function to be `_get_collection` because list confused me a little bit. So get resource returns an item and get collection returns a list, no need to comment out the `assert len(rsp) == 1` in `_get_resource`

Reply:
done